_redirects

```diff
@@ -1,30 +1,117 @@
-/articles/can-multi-year-ssl-certificate                  /articles/can-multi-year-ssl-certificates/
-/articles/can-multi-year-ssl-certificate/index.html       /articles/can-multi-year-ssl-certificates/
-/articles/secondary-dns-support                           /articles/secondary-dns/
-/articles/secondary-dns-support/index.html                /articles/secondary-dns/
-/articles/advanced-editor                                 /articles/record-editor/
-/articles/advanced-editor/index.html                      /articles/record-editor/
-/articles/moving-domain                                   /articles/transferring-domain-between-accounts/
-/articles/moving-domain/index.html                        /articles/transferring-domain-between-accounts/
-/articles/url-redirect-ssl                                /articles/redirector/
-/articles/url-redirect-ssl/index.html                     /articles/redirector/
-/articles/redirector-https/                               /articles/redirector/
-/articles/redirect/                                       /articles/redirector/
-/articles/what-ssl-certificate-authorities                /articles/ssl-certificate-authorities/
-/articles/what-ssl-certificate-authorities/index.html     /articles/ssl-certificate-authorities/
-/articles/renewing-ssl-certificates                       /articles/renewing-ssl-certificate/
-/articles/renewing-ssl-certificates/index.html            /articles/renewing-ssl-certificate/
-/articles/reissuing-ssl-certificates                      /articles/reissuing-ssl-certificate/
-/articles/reissuing-ssl-certificates/index.html           /articles/reissuing-ssl-certificate/
-/articles/ssl-certificate-hostname                        /articles/ssl-certificate-names/
-/articles/ssl-certificate-hostname/index.html             /articles/ssl-certificate-names/
-/articles/master-slave-support                            /articles/secondary-dnsimple/
-/articles/master-slave-support/index.html                 /articles/secondary-dnsimple/
-/articles/purchasing-ssl-certificates                     /articles/ordering-standard-certificate/
-/articles/purchasing-ssl-certificates/index.html          /articles/ordering-standard-certificate/
-/articles/ordering-comodo-certificate                     /articles/ordering-standard-certificate/
-/articles/ordering-comodo-certificate/index.html          /articles/ordering-standard-certificate/
-/articles/enabling-and-disabling-auto-renewal             /articles/domain-auto-renewal/
-/articles/enabling-and-disabling-auto-renewal/index.html  /articles/domain-auto-renewal/
-/articles/dnssec-support                                  /articles/dnssec/
-/articles/dnssec-support/index.html                       /articles/dnssec/
+/articles)/ssl-certificate-hostname                                       /articles/can-upgrade-single-hostname-ssl-certificate-to-wildcard/
```

Comment:
Is this supposed to have the `)`

Reply:
This is on the spreadsheet as linked above, which I believe was proved by Enrico. This might be a reasonable one to leave out, as it is clearly miswritten. Do you think we should _can_ it?

Reply:
The route is listed in the spreadsheet with and without the parenthesis. I would keep both for now.